DiffusionPreprocessing/DiffPreprocPipeline_Split.sh

```diff
@@ -0,0 +1,390 @@
+#!/bin/bash
+#~ND~FORMAT~MARKDOWN~
+#~ND~START~
+#
+# # DiffPreprocPipeline.sh
```

Comment:
Script name here, and elsewhere in this section needs to be updated.

Reply:
In fact, this entire markdown section probably needs to be updated to be relevant for this specific script.